servatrice/servatrice.ini.example

```diff
@@ -99,6 +99,15 @@ allowedpunctuation=_.-
 ; If a username can begin with punctuation defined in allowedpunctuation
 allowpunctuationprefix=false
 
+; Enable forgot password functionality
+enableForgotPassword=true
+
+; Validate users inputed email matches email on account before allowing forgot password reset
+forgotpasswordemailrequired=true
+
+; Validate users clientid matches clientid on account before allowing forgot password reset
+forgotpasswordclientidrequired=true
```

Comment:
Many people tend to forget their passwords after not using the service for some time... so it might happen that they change their hardware in between.

They won't be able to reset the password and make the server send a new one to their linked mail address?